subgraphs/products/products.graphql

```diff
@@ -1,6 +1,13 @@
 extend schema
     @link(url: "https://specs.apollo.dev/federation/v2.0",
```

Comment:
@korinne should `subgraph-js` require Fed v2.1 for use of `@composeDirective` like `federation-jvm` does?